File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/models/Country.java
package com.eulersbridge.isegoria.network.api.models;

import com.squareup.moshi.Json;

import java.util.List;

public class Country {

	@Json(name = "countryId")
	public long id;

	@Json(name = "countryName")
	public String name;
	public List<Institution> institutions;

	@Override
	public String toString() {
		return name;
	}
}

File: isegoria/src/main/java/com/eulersbridge/isegoria/network/api/models/Country.kt
package com.eulersbridge.isegoria.network.api.models

import com.squareup.moshi.Json

data class Country(
    @Json(name = "countryId")
    val id: Long = 0,

    @Json(name = "countryName")
    val name: String,

    val institutions: List<Institution>?
) {
    override fun toString(): String {
        return name
    }
}